packages/celery-library/README.md

```diff
@@ -0,0 +1,17 @@
+# simcore Celery library
+
+Provides a wrapper around Celery library.
```

Comment:
MINOR: http link to celery

Provides a wrapper functionality around https://github.com/celery/celery